trinity/main.py

```diff
@@ -171,6 +174,8 @@ def main() -> None:
             console(chain_config.jsonrpc_ipc_path, use_ipython=not args.vanilla_shell)
         else:
             networking_process.join()
+    except FileNotFoundError:
```

Comment:
Can you wrap the `console(...)` invocation in one common method that catches and does the `sys.exit(1)`